### PR TITLE
DAOS-623 java: Update log4j-core version due to security alert (#3027)

### PR DESCRIPTION
**Pull request by jolivier23**
Update to version 2.13.3 or greater due to github security alert

Also, we need to use a fixed version of fio in the SPDK build to ensure it can always build.

Signed-off-by: Jeff Olivier <jeffrey.v.olivier@intel.com>

### CODE REVIEW

File: utils/build.config [hunk 1]
```diff
@@ -8,6 +8,7 @@ PMDK = 1.8
 ISAL = v2.26.0
 SPDK = v19.04.1
 FUSE = fuse-3.6.1
+FIO = fio-3.20
```
Comment (brianjmurrell) [inline utils/build.config:11]: Why is this needed now and wasn't when 1.0 was created?

Comment (jolivier23) [reply utils/build.config:11]: It was definitely needed as proved by the fact that it doesn't build now.  The fact that it wasn't there was a bug.

Note that fio isn't used in master anyway so this is just a workaround for anyone that checks out 1.0

Comment (brianjmurrell) [reply utils/build.config:11]: Yeah, I knew about the removal in master.

I'm still puzzled how any of, say, the 1.0.0 RCs built if this was needed.  Maybe I will find a minute to investigate.

Comment (brianjmurrell) [reply utils/build.config:11]: So it seems that 1.0.0rc2 (for example) did actually checkout and build fio, even though it was missing from `build.config`.  It must have just build the default checkout branch (probably master) after the clone and ended up building:

```
FIO_VERSION = fio-3.20-38-g14060
```

Comment (jolivier23) [reply utils/build.config:11]: yeah, I think there has been a regression in fio master since we built it.   It failed my build before I added the 2nd commit
